audiocall/src/main/java/com/voximplant/demos/kotlin/audio_call/services/AudioCallManagerDefault.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2011 - 2024, Zingaya, Inc. All rights reserved.
+ */
+
+package com.voximplant.demos.kotlin.audio_call.services
+
+import android.content.Context
+import com.voximplant.demos.kotlin.utils.*
+import com.voximplant.sdk.call.*
+import com.voximplant.sdk.client.IClient
+import java.util.*
+
+class AudioCallManagerDefault(
+    context: Context,
+    client: IClient,
+) : AudioCallManager(context, client) {
+
+    override fun onIncomingCall(call: ICall, video: Boolean, headers: Map<String?, String?>?) {
+        super.onIncomingCall(call, video, headers)
+        showIncomingCallUI()
+    }
+
+    @Throws(CallManagerException::class)
+    override fun startOutgoingCall() = executeOrThrow {
+        super.startOutgoingCall()
+        managedCall?.start() ?: throw noActiveCallError
```

Comment:
I think it should be in another order, since we should ensure that the call is started (i.e. the exception is not thrown) and only after that update the state to connecting

Reply:
In this case, it is necessary to update the AudioCallManagerTelecom logic.

Reply:
Updated